### PR TITLE
Add action to deploy github pages

### PR DESCRIPTION
**Pull request by avidit**
This will add action to deploy the app bundled with production notification url to github pages

### CODE REVIEW

File: .github/workflows/pages.yml [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+  pages: write
+  id-token: write
+
+# Allow one concurrent deployment
+concurrency:
+  group: "pages"
+  cancel-in-progress: true
+
+env:
+  NOTIFICATION_URL: https://ddehnr4ewobxc.cloudfront.net/dynNotifications.json
```
Comment (zeusongit) [inline .github/workflows/pages.yml:23]: Can we use the URL from env files here? Just to keep a single source of truth?
Also why are deploying on gh-pages?

Comment (avidit) [reply .github/workflows/pages.yml]: Removed env from workflow file.

**Comment by QilongTang** [comment]
@avidit I think I missed this PR, what is the purpose of it?

**Comment by avidit** [comment]
> @avidit I think I missed this PR, what is the purpose of it?

This will deploy the notification center as github pages. We can use this as one more point of validation.
e.g https://avidit.github.io/NotificationCenter/

**Pull request closed without merging**